src/asm/rgbasm.1

```diff
@@ -7,89 +7,187 @@
 .\"
 .Dd July 8, 2019
 .Dt RGBASM 1
-.Os RGBDS Manual
+.Os
 .Sh NAME
 .Nm rgbasm
 .Nd Game Boy assembler
 .Sh SYNOPSIS
-.Nm rgbasm
-.Op Fl EhLVvw
+.Nm
 .Op Fl b Ar chars
 .Op Fl D Ar name Ns Op = Ns Ar value
+.Op Fl E
 .Op Fl g Ar chars
+.Op Fl h
 .Op Fl i Ar path
+.Op Fl L
 .Op Fl M Ar dependfile
 .Op Fl o Ar outfile
 .Op Fl p Ar pad_value
 .Op Fl r Ar recursion_depth
-.Ar file
+.Op Fl V
+.Op Fl v
+.Op Fl w
+.Ar
 .Sh DESCRIPTION
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates a RGB object file from an assembly source file.
```

Comment:
“an RGB”, IMO. (But I don’t feel strongly about it.)

Reply:
Fine by me too. Will fix as suggested.